taxi/forms.py

```diff
@@ -0,0 +1,45 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if len(license_number) != 8:
+            raise forms.ValidationError(
+                "License number must be exactly 8 characters long.")
+
+        if (not license_number[:3].isupper()
+                or not license_number[:3].isalpha()):
+            raise forms.ValidationError(
+                "The first three characters must be uppercase letters.")
+
+        if not license_number[-5:].isdigit():
+            raise forms.ValidationError(
+                "The last five characters must be digits.")
+
+        return license_number
+
+
+class DriverCreateForm(DriverLicenseUpdateForm, UserCreationForm):
+    class Meta:
+        model = Driver
+        fields = (UserCreationForm.Meta.fields
+                  + ("first_name", "last_name", "email", "license_number",))
+
+
+class CarCreateForm(forms.ModelForm):
+    drivers = forms.ModelChoiceField(queryset=get_user_model().objects.all(),
+                                     widget=forms.CheckboxSelectMultiple(),
+                                     required=False)
```

Comment:
The `drivers` field is defined as a `ModelChoiceField` but uses `CheckboxSelectMultiple` as its widget. This combination is incorrect because `ModelChoiceField` is meant for single selections. Consider using `ModelMultipleChoiceField` if multiple selections are intended.